src/js/components/Video/Video.js

```diff
@@ -254,10 +274,71 @@ const Video = forwardRef(
         onClick: () => showCaptions(caption.active ? -1 : 0),
       }));
 
+      const buttonTable = {
+        closedCaption: captionControls,
+        fullScreen: {
+          onClick: fullscreen,
+        },
+        pause: {
+          onClick: playing ? pause : play,
+        },
+        play: {
+          onClick: playing ? pause : play,
+        },
+        reduceVolume: {
+          a11yTitle: 'video.volumeDown',
+          onClick: volume >= VOLUME_STEP ? quieter : undefined,
+          close: false,
+        },
+        volume: {
+          a11yTitle: 'video.volumeUp',
+          onClick: volume <= 1 - VOLUME_STEP ? louder : undefined,
+          close: false,
+        },
+      };
+
+      let newItem;
+      const controlsArr =
+        newControls.items &&
+        newControls.items.map((item) => {
+          if (typeof item === 'string') {
+            // Transforming item into PascalCase for lookup on Icons
+            const newString = item.charAt(0).toUpperCase() + item.slice(1);
```

Comment:
This feels a little fragile to me. Can we instead create an object to map them formally at the top of this file?